modules/ROOT/pages/jakarta-ee10-diff.adoc

```diff
@@ -134,10 +135,138 @@ If this attribute set to `false`, which is the default, then an archive that con
 [#restfulws]
 == Differences between Jakarta RESTful Web Services 3.1 and 3.0
 
+[#servlet]
+== Differences between Jakarta Servlet 6.0 and 5.0
+
+The feature:servlet-6.0[display=Jakarta Servlet 6.0] feature provides new APIs and new configurations to set any attribute on a response cookie.
+
+=== New ways to configure cookie attributes
+
+In Jakarta Servlet 6.0, you can set attributes on a response cookie either a deployment descriptor file, programmatically with the `jakarta.servlet.ServletContainerInitializer` API, or at request time.
+
+You can set cookie attributes in a `web.xml` deployment descriptior file by specifying the `attribute` element within the `cookie-config` element, as shown in the following example.
+
+[source,xml]
+----
+<session-config>
+   <cookie-config>
+     <name>SampleCookieName</name>
+     <domain>my.domain.com</domain>
+     <path>/</path>
+     <http-only>true</http-only>
+     <secure>true</secure>
+     <attribute>
+        <attribute-name>MyAttributeName</attribute-name>
+        <attribute-value>valueExample</attribute-value>
+     </attribute>
+     <attribute>
+        <attribute-name>MyOtherAttribute</attribute-name>
+        <attribute-value>OtherValue</attribute-value>
+     </attribute>
+   </cookie-config>
+</session-config>
+----
+
+You can also set cookie attributes dynamically from an application during pplication startup by using the `jakarta.servlet.ServletContainerInitializer`, as shown in the following example.
+
+[source,java]
+----
+@Override
+public void onStartup(Set<Class<?>> setOfClasses, ServletContext context) throws ServletException {
+      SessionCookieConfig scc = context.getSessionCookieConfig();
+      scc.setName("MyDynaCookieName");
+      scc.setAttribute("MyDynamicAttribute", "DynamicValue");
+      ..........
+}
+----
+
+Alternatively, you can set attributes from the application at request time, as shown in the following example.
+
+[source,java]
+----
+SessionCookieConfig scc = getServletContext().getSessionCookieConfig();
+scc.setName("MyRequestCookieName");
+scc.setAttribute("MyRequestAttribute", "RequestValue");
+----
+
+If conflicts arise among different sources for the cookie configuration, the following precedence order applies, from lowest to highest precedence, to resolve the conflict.
```

Comment:
@dmuelle  should this be "highest to lowest precedence" ?  ( the request time has highest priority; web.xml has lowest priority)

Reply:
thanks for the catch- I misread this in the issue comment. will fix